raiden/raiden_service.py

```diff
@@ -19,19 +20,26 @@
 )
 from raiden.blockchain.abi import CHANNEL_MANAGER_ABI, REGISTRY_ABI
 from raiden.network.channelgraph import ChannelGraph
-from raiden.tasks import AlarmTask, StartExchangeTask, HealthcheckTask
+from raiden.tasks import AlarmTask, LogListenerTask, StartExchangeTask, HealthcheckTask
 from raiden.encoding import messages
 from raiden.messages import SignedMessage
 from raiden.network.protocol import RaidenProtocol
-from raiden.utils import privatekey_to_address, isaddress, pex, GLOBAL_CTX
+from raiden.utils import privatekey_to_address, isaddress, pex, GLOBAL_CTX, camel_to_snake_case
+
+from raiden.api.objects import (
+    TransferReceived,
+    ChannelClosed,
+    ChannelNew,
+    ChannelNewBalance,
+    ChannelSecretRevealed,
+    ChannelSettled
+)
 
 log = slogging.get_logger(__name__)  # pylint: disable=invalid-name
 
 DEFAULT_REVEAL_TIMEOUT = 30
 DEFAULT_SETTLE_TIMEOUT = DEFAULT_REVEAL_TIMEOUT * 20
 
-EventListener = namedtuple('EventListener', ('event_name', 'filter_', 'translator'))
```

Comment:
**Careful**: this didn't make it in my pull-rebase from origin/master (somehow).
If I cannot rewind this easily, it has to be cherrypicked and reimplemented again!